float32[] voltage_output
